src/h5web/App.test.tsx

```diff
@@ -126,9 +126,115 @@ test('navigate groups in explorer', async () => {
 
   expect(queryByRole('treeitem', { name: '1_cormap' })).not.toBeInTheDocument();
 
-  // `waitFor` avoids warning due to async work occuring after completion of test
+  // `waitFor` avoids warning due to async work occurring after completion of test
   await waitFor(() => {
     expect(groupBtn).toHaveAttribute('aria-selected', 'true');
     expect(groupBtn).toHaveAttribute('aria-expanded', 'false');
   });
 });
+
+test('select a scalar dataset and display the value', async () => {
+  const { findByRole, getByRole, getByText } = renderApp();
+
+  // Select the scalar dataset entry_0000/1_cormap/version
+  fireEvent.click(await findByRole('treeitem', { name: 'entry_0000' }));
+  fireEvent.click(getByRole('treeitem', { name: '1_cormap' }));
+  const datasetBtn = getByRole('treeitem', { name: 'version' });
+  expect(datasetBtn).toBeVisible();
+  fireEvent.click(datasetBtn);
```

Comment:
Some blank lines would make things more readable -- around (groups of) simulated events especially.

Reply:
Actually, it might make sense to abstract this into a utility function, what do you think? Something like `navigateTo('entry_0000/1_cormap/version')` - or perhaps with an array.